conode/test.sh

```diff
@@ -21,6 +22,23 @@ testConode(){
     testFail runCo 1 check -g public.toml
 }
 
+testDatabase(){
+    runCoBG 1
+    sleep .1
```

Comment:
Is this needed? I think that `runCoBG` checks whether the conode started correctly. Same below.